internal/feedback-tool/src/main/java/org/wso2/security/tool/util/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.security.tool.util;
+
+/**
+ * Constants -- This utility class consists of all the collected constant values of the project.
+ * All the members of this class are immutable.
+ *
+ * @author Arshika Mohottige
+ */
+public final class Constants {
+    public static final String OUTPUT_HTML_FILE = "output.html";
+    public static final String OUTPUT_PDF_FILE = "output.pdf";
+    public static final String JSON_DATA_OBJECT = "data";
+    public static final String PROPERTIES_FILE_PATH = "resource/conf.properties";
```

Comment:
Better to add a file separator otherwise this path would become dependent on the OS.